include/cppflow/model.h

```diff
@@ -84,6 +85,12 @@ namespace cppflow {
 
         std::vector<TF_Output> inp_ops(inputs.size());
         std::vector<TF_Tensor*> inp_val(inputs.size());
+
```

Comment:
It's difficult to do that for `unique_ptr`.

`std::vecotr<std::unique_ptr<TF_Tenosr, decltype(&TF_DeleteTensor)>> ` is not compitiable for `TF_SessionRun` unless you copy the raw pointer to a new vector. It's not convinient.

defer is a general RAII class for any resource management. It's really useful in our production environment.